resources/inputmaps/apple-media-keys.json

```diff
@@ -3,7 +3,9 @@
   "idmatcher": "AppleMediaKeys.*",
   "mapping":
   {
-    "KEY_PLAY": "play_pause",
+    "KEY_PLAY": "play",
+    "KEY_PAUSE": "pause",
+    "KEY_PLAY_PAUSE": "play_pause",
```

Comment:
I mean, I'm worried about existing user input mappings. Will they break with this change? It sounds like they possibly could. Then again, I'm not too worried.